content/conditions/tendonitis/content-3.md

```diff
@@ -1 +1 @@
-[Find a pharmacy]({{ findersBaseUrl }}find-help)
\ No newline at end of file
+[Find a pharmacy]({{ findersBaseUrl }}finders/find-help)
```

Comment:
Given that the route cause of the bug is that the content editor got the link wrong or didn't update when it changed then I think that `findersBaseUrl` should be include `finders/find-help` or if we still need to preserve `findersBaseUrl` we can add `findersUrl` which is defined once.